cmd/serv.go

```diff
@@ -238,7 +238,8 @@ func runServ(c *cli.Context) error {
 		if git.DefaultFeatures().SupportProcReceive {
 			// for AGit Flow
 			if cmd == "ssh_info" {
-				fmt.Print(`{"type":"gitea","version":1}`)
+				data := private.GetSSHInfo(ctx)
+				fmt.Print(data)
```

Comment:
```suggestion
				fmt.Println(data)
```

Reply:
Actually, it should `c *cli.Context` stdout, to make it testable.